### PR TITLE
Peers logs improvement

### PR DESCRIPTION
**Pull request by nickeskov**

### CODE REVIEW

File: pkg/node/peer_manager/peer_manager.go [hunk 1]
```diff
@@ -109,27 +109,28 @@ func (a *PeerManagerImpl) NewConnection(p peer.Peer) error {
...
 		err := errors.Errorf(
-			"versions are too different, current %s, connected %s",
+			"versions are too different, current %s, connected %s (peer '%s')",
 			a.version.String(),
 			p.Handshake().Version.String(),
+			p.ID(),
 		)
 		a.restrict(p, now, err.Error())
 		_ = p.Close()
-		return proto.NewInfoMsg(err)
```
Comment (alexeykiselev) [inline pkg/node/peer_manager/peer_manager.go:118]: Why move this error logging from DEBUG to ERROR level?

File: pkg/node/peer_manager/peer_manager.go [hunk 2]
```diff
@@ -109,27 +109,28 @@ func (a *PeerManagerImpl) NewConnection(p peer.Peer) error {
...
+		return err
 	}
 	if p.Handshake().AppName != a.networkName {
 		err := errors.Errorf("peer '%s' has the invalid network name '%s', required '%s'",
 			p.ID(), p.Handshake().AppName, a.networkName)
 		a.restrict(p, now, err.Error())
 		_ = p.Close()
-		return proto.NewInfoMsg(err)
```
Comment (alexeykiselev) [inline pkg/node/peer_manager/peer_manager.go:125]: And this one. Those errors are very common and they will bloat the log with error messages. Has meaning only on DEBUG level.

**Review by alexeykiselev — CHANGES_REQUESTED** [review]

File: pkg/node/peer_manager/peer_manager.go (hunk 1 above)
Comment (nickeskov) [reply pkg/node/peer_manager/peer_manager.go]: That shouldn't be there. I forgot to return it back. Fixed.

File: pkg/node/peer_manager/peer_manager.go (hunk 2 above)
Comment (nickeskov) [reply pkg/node/peer_manager/peer_manager.go]: The same as above, fixed.

**Review by alexeykiselev — APPROVED** [review]

**Pull request merged**